### PR TITLE
VSF example

### PR DESCRIPTION
**Pull request by satelllte**

### CODE REVIEW

**Comment by tristanlee85** [comment]
@egormesyats is this public in the core API? https://github.com/moovweb-docs/xdn-examples/blob/vsf-example/xdn-vue-storefront-example/xdn/routes.js#L3 This looks to be an internal constant of the framework and could be confusing to someone looking through the example.

File: xdn-vue-storefront-example/xdn/routes.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
+import { Router } from '@xdn/core/router'
+import { CACHE_ASSETS, CACHE_PAGES } from './cache'
+import { BACKENDS } from '@xdn/core/constants'
```
Comment (tristanlee85) [inline xdn-vue-storefront-example/xdn/routes.js]: @egormesyats is this public in the core API? This looks to be an internal constant of the framework and could be confusing to someone looking through the example.

Comment (satelllte) [reply xdn-vue-storefront-example/xdn/routes.js]: Yes. It's documented here: https://developer.moovweb.com/guides/react#section_configuring_the_xdn_router

File: xdn-vue-storefront-example/xdn/routes.js [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+  router.get(page, ({ cache, proxy }) => {
+    cache(CACHE_PAGES)
+    proxy(BACKENDS.js)
+  })
+})
+
+// fallback
+router.fallback(({ proxy }) => {
+  proxy(BACKENDS.js)
+})
```
Comment (tristanlee85) [inline xdn-vue-storefront-example/xdn/routes.js]:
```suggestion
router.fallback(({ renderWithApp }) => {
    renderWithApp()
  })
```

File: xdn-vue-storefront-example/xdn/routes.js [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+  cache(CACHE_PAGES)
+  proxy('origin')
+})
+
+// pages
+pages.forEach(page => {
+  router.get(page, ({ cache, proxy }) => {
+    cache(CACHE_PAGES)
+    proxy(BACKENDS.js)
+  })
```
Comment (tristanlee85) [inline xdn-vue-storefront-example/xdn/routes.js]:
```suggestion
  router.get(page, ({ cache, renderWithApp }) => {
    cache(CACHE_PAGES)
    renderWithApp()
  })
```

File: xdn-vue-storefront-example/xdn/routes.js (hunk 1 above)
Comment (tristanlee85) [reply xdn-vue-storefront-example/xdn/routes.js]:
```suggestion
```

**Review by tristanlee85 — CHANGES_REQUESTED** [review]

**Pull request merged**